src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ReportService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String HEADER_LINE = "fruit,quantity";
+    private static final String COMMA = ",";
```

Comment:
```suggestion
    private static final String SEPARATOR = ",";
```
if you ever will need to change the constant to something other than comma, you will need to change both constant name and value